libs/designsystem/src/lib/components/data-table/table/table.component.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  ChangeDetectorRef,
+  Component,
+  HostBinding,
+  Input,
+  OnInit,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'table[kirby-table]',
+  template: `<ng-content></ng-content>`,
+  styleUrls: ['./table.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class TableComponent implements OnInit {
+  @HostBinding('class.kirby-table-layout-fixed') @Input() fixedLayout: boolean = false;
+
+  constructor(private readonly cdRef: ChangeDetectorRef) {}
+
+  ngOnInit(): void {
+    this.cdRef.detach();
```

Comment:
I don't know if it's a known pattern, I chose it because I read an article about changeDetection - I think in it's early stages - and that was the suggested approach. But i changed it :)